tools/functional-tester/etcd-tester/failure.go

```diff
@@ -136,26 +136,29 @@ func (f *failureUntilSnapshot) Inject(c *cluster, round int) error {
 	if err := f.failure.Inject(c, round); err != nil {
 		return err
 	}
-
 	if c.Size < 3 {
 		return nil
 	}
-
-	start, _ := c.Report()
-	end := start
+	startRev, err := c.maxRev()
```

Comment:
this might cause problem i think. we just injected failures at line 136. if we check maxRev now, we should expect some failures. if we kill all members, we cannot even find the any rev.